### PR TITLE
Fix: Updated Harsh Patel's photo

### PR DESCRIPTION
**Pull request by msynowski**
Fix of #560

### CODE REVIEW

**Review by atm1504 — APPROVED** [review]

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-fossasia.org
```
Comment (liveHarshit) [inline CNAME]: Why?

**Review by liveHarshit — CHANGES_REQUESTED** [review]
.

**Comment by Shizerq** [comment]
Had to set up page on my own gh pages as it was mentioned in GCI task

**Comment by msynowski** [comment]
Had to set up page on my own gh pages as it was mentioned in GCI task

**Comment by msynowski** [comment]
If it’s a problem I can revert this commit and setup gh pages on other branch

File: CNAME (hunk 1 above)
Comment (liveHarshit) [reply CNAME]: Revert it and provide preview URL with gh pages.

**Comment by kushthedude** [comment]
> If it’s a problem I can revert this commit and setup gh pages on other branch

Revert this and while enabling with gh-pages, CNAME won't give you error on the fork.

File: CNAME (hunk 1 above)
Comment (uds5501) [reply CNAME]: Also, @Shizerq Try and squash your commits.

**Review by liveHarshit — DISMISSED** [review]

**Comment by liveHarshit** [comment]
Provide preview url with gh pages as per the task.

**Review by kushthedude — CHANGES_REQUESTED** [review]
Please squash your commits

**Comment by msynowski** [comment]
Squashed the commits and reverted the CNAME change

**Review by nikit19 — APPROVED** [review]

**Comment by liveHarshit** [comment]
> Please squash your commits

They are high school students, it is okay to make multiple commits. We can squash and merge them.

**Comment by msynowski** [comment]
https://shizerq.github.io/fossasia.org/team/

**Review by liveHarshit — APPROVED** [review]

**Pull request merged**